dev/TabView/TabView.xaml

```diff
@@ -43,20 +43,35 @@
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:TabView">
-                    <Grid x:Name="TabViewContainer">
+                    <Grid>
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="AddButtonStates">
+                                <VisualState x:Name="AddButtonVisible"/>
+                                <VisualState x:Name="AddButtonCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="AddButton.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+
                         <Grid.RowDefinitions>
                             <RowDefinition Height="Auto" />
                             <RowDefinition Height="*" />
                         </Grid.RowDefinitions>
 
-                        <Grid>
+                        <Grid x:Name="TabContainerGrid"
+                            Background="{ThemeResource TabViewBackground}">
                             <Grid.ColumnDefinitions>
-                                <ColumnDefinition Width="Auto"/>
-                                <ColumnDefinition Width="*"/>
-                                <ColumnDefinition Width="Auto"/>
+                                <ColumnDefinition Width="Auto" x:Name="LeftContentColumn"/>
```

Comment:
>LeftContentColumn [](start = 71, length = 17)

Why name these ColumnDefinitions now?

Reply:
Because I need to load them in code to subtract their sizes from available space.

Reply:
This feels like it might be kinda like TemplateSettings - is that a better approach here?

Reply:
Can you be more specific on how that would work?

Reply:
Thinking about this more, maybe this really points to a desire to have a TabPanel of some kind (WPF's TabControl has this) which can have this layout logic, instead of named columns that need to be in the template.

Reply:
I may need to. So look, I'm about to make TabView contain a ListView instead of being a ListView, so all this logic is about to change anyway... I'll make it cleaner in the next version.

Reply:
Ah, gotcha

Reply:
Alas. :)